spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 ENV['RAILS_ENV'] ||= 'test'
 
+require "coveralls"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.